app/views/jobseekers/job_applications/build/_job_application_steps.html.slim

```diff
@@ -0,0 +1,9 @@
+= render StepsComponent.new title: "Application Steps", classes: "govuk-!-margin-top-6" do |c|
+  - c.step(label: "Personal Details", current: current_step?(:personal_details), completed: wizard_steps[0].to_s.in?(job_application.completed_steps))
+  - c.step(label: "Professional Status", current: current_step?(:professional_status), completed: wizard_steps[1].to_s.in?(job_application.completed_steps))
+  - c.step(label: "Employment History", current: current_step?(:employment_history), completed: wizard_steps[2].to_s.in?(job_application.completed_steps))
+  - c.step(label: "Personal Statement", current: current_step?(:personal_statement), completed: wizard_steps[3].to_s.in?(job_application.completed_steps))
+  - c.step(label: "References", current: current_step?(:references), completed: wizard_steps[4].to_s.in?(job_application.completed_steps))
+  - c.step(label: "Equal Opportunities", current: current_step?(:equal_opportunities), completed: wizard_steps[5].to_s.in?(job_application.completed_steps))
+  - c.step(label: "Ask for Support", current: current_step?(:ask_for_support), completed: wizard_steps[6].to_s.in?(job_application.completed_steps))
+  - c.step(label: "Declarations", current: current_step?(:declarations), completed: wizard_steps[7].to_s.in?(job_application.completed_steps))
```

Comment:
You could refactor this to something like:

```ruby
= render StepsComponent.new title: t("jobseekers.job_applications.process_title", classes: "govuk-!-margin-top-6" do |c|
  - wizard_steps.each_with_index do |step, index|
    - c.step(label: t("jobseekers.job_applications.build.#{step}.heading"), current: current_step?(step), completed: wizard_steps[index].to_s.in?(job_application.completed_steps))
```

Reply:
I forgot to add the translations! Cheers :)

Reply:
It only needs an each too, because the completed boolean should just be 
```ruby
step.to_s.in?(job_application.completed_steps
```